docs/table_design/Data_distribution.md

```diff
@@ -547,6 +547,10 @@ Buckets reflect how data files are actually organized in StarRocks.
 
     To enable this feature, make sure that the FE dynamic parameter `enable_auto_tablet_distribution` is set to `TRUE`. After a table is created, you can execute [SHOW CREATE TABLE](../sql-reference/sql-statements/data-manipulation/SHOW%20CREATE%20VIEW.md) to view the bucket number automatically set by StarRocks.
 
+    > **NOTICE**
+    >
+    > If the raw data size of a partition exceeds 100GB, we recommend that you manually configure the number of buckets using the following method.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/4ca3bf15-4c41-494a-a281-7d2889d785a9)
1）-> 100 MB
2) -> the following Method 2 OR Method 2 described below. (I think "Method 2" is easier to find.)